frontend/app/src/components/inputs/RevealableInput.vue

```diff
@@ -14,8 +13,13 @@
     :error-messages="errorMessages"
     v-on="$listeners"
     @input="input"
-    @click:append="revealed = !revealed"
-  />
+  >
```

Comment:
Proably this needs a `/`?

Reply:
No, now it has the template inside so it is not self-closing. it has a closing tag